src/components/users/users.ts

```diff
@@ -151,6 +172,27 @@ async function setAllUserRolesForOrg(
   ));
 }
 
+export function _getUserRolesByGuid(
+    userOrgRoles: ReadonlyArray<IOrganizationUserRoles>,
+    spaceUserLists: ReadonlyArray<ISpaceUsers>,
+  ): IUserRolesByGuid {
+
+  const spacesByUser =
```

Comment:
during testing it came out, that this map contains maximum one space for any user

Reply:
I've addressed this now. This case is covered by unit tests now, and I've tested it in my dev environment.